### PR TITLE
Feature/search

### PR DESCRIPTION
**Pull request by Mamatisaev**

### CODE REVIEW

File: src/main/java/kg/peaksoft/taskTrackerb6/db/service/UserService.java [hunk 1]
```diff
@@ -161,7 +164,7 @@ public AuthResponse authWithGoogle(String tokenId) throws FirebaseAuthException
             newUser.setRole(Role.ADMIN);
             user = repository.save(newUser);
         }
-        user = repository.findByEmail(firebaseToken.getEmail()).orElseThrow(
+        user = repository.findUserByEmail(firebaseToken.getEmail()).orElseThrow(
                 () -> new NotFoundException("user with this email not found!"));
         String token = jwtUtil.generateToken(user.getPassword());
```
Comment (datka-mcoder) [inline src/main/java/kg/peaksoft/taskTrackerb6/db/service/UserService.java:169]: Generete token with user email

File: src/main/java/kg/peaksoft/taskTrackerb6/api/AuthApi.java [hunk 2]
```diff
@@ -61,4 +57,10 @@ public ResetPasswordResponse resetPassword(@RequestBody @Valid ResetPasswordRequ
     public AuthResponse authWithGoogleAccount(@RequestParam String token) throws FirebaseAuthException {
         return userService.authWithGoogle(token);
     }
+
+    @Operation(summary = "Search users", description = "Search users by first/last name & e-mail")
```
Comment (i-Coderr) [inline src/main/java/kg/peaksoft/taskTrackerb6/api/AuthApi.java:61]: Pls write in summary = Global search, description = Search by user email, first and last name

**Review by i-Coderr — COMMENTED** [review]
Commented at 3.41 17/11/22

**Comment by i-Coderr** [comment]
Well done!

**Review by i-Coderr — COMMENTED** [review]
Well done!

**Pull request merged**